.github/codejson/{{cookiecutter.project_name}}/code.json

```diff
@@ -1,49 +1,38 @@
 {
-  "name": "{{ cookiecutter.project_name }}",
+  "project_name": "{{ cookiecutter.project_name }}",
+  "project_repo_name": "{{ cookiecutter.project_repo_name }}",
+  "project_org": "{{ cookiecutter.project_org }}",
   "description": "{{ cookiecutter.description }}",
-  "longDescription": "{{ cookiecutter.long_description }}",
   "status": "{{ cookiecutter.status }}",
   "permissions": {
     "licenses": [
       {
-        "URL": "LICENSE",
-        "name": "{{ cookiecutter.license }}"
+        "license": "{{ cookiecutter.license }}",
```

Comment:
this field should be named `name` according to the schema: https://github.com/DSACMS/gov-codejson/blob/main/schemas/schema-2.0.0.json#L42-L62